gum/backend-darwin/gumkernel-darwin.c

```diff
@@ -698,13 +700,22 @@ gum_kernel_do_find_base_address (void)
   float version;
   GumAddress base = 0;
 
-  version = gum_kernel_get_version ();
-
 #ifdef HAVE_ARM64
-  if (version >= 16.0) /* iOS 10.0+ */
-    base = gum_kernel_bruteforce_base (G_GUINT64_CONSTANT (0xfffffff007004000));
-  else if (version >= 15.0) /* iOS 9.0+ */
-    base = gum_kernel_bruteforce_base (G_GUINT64_CONSTANT (0xffffff8004004000));
+  base = gum_kernel_get_base_from_all_image_info ();
+  if (base == 0)
+  {
+    version = gum_kernel_get_version ();
+    if (version >= 16.0) // iOS 10.0+
```

Comment:
```suggestion
    if (version >= 16.0) /* iOS 10.0+ */
```

Should stick to C-style comments for consistency.